subworkflows/ensembl/prepare_genome_metadata/meta.yml

```diff
@@ -0,0 +1,39 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/subworkflows/yaml-schema.json
```

Comment:
```suggestion
```

Reply:
So I noticed, we were removing these lines early on. I believe vasily wanted to remove them. nf-core create modules inserts these lines, while also being retained in published nf-core modules. We will need to make a choice, remove all or retroactively reinsert to ported modules

Reply:
personally I do not like it too much but we can decide on this and change the modules later.